spec/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Noticed this is eslint 1 - maybe worth getting it in and then doing this and updating to eslint v2 in a seperate PR